Cargo.toml

```diff
@@ -86,16 +90,26 @@ swap = [
   "nightly",
   "memmap2",
 ]
+std_fs = []
 
 # Shortcuts
-all_except_failpoints = ["internals", "scripting", "nightly", "swap"]
-all_stable = ["internals", "scripting", "failpoints"]
-all_stable_except_failpoints = ["internals", "scripting"]
+all_unix_stable_without_failpoints = ["internals", "scripting"]
```

Comment:
These look a bit messy. I'm cleaning them up in https://github.com/tikv/raft-engine/pull/325. Once it is merged you can add std_fs to Makefile `test_matrix` without polluting other build options.